pop-cli-for-appchains/README.md

```diff
@@ -21,6 +21,7 @@ With Pop CLI, developers can build an appchain on Polkadot.
 * Spin up a Polkadot network with various parachains
   * Useful for testing cross-chain interoperability
 * Benchmarking of nodes and runtimes&#x20;
+* Testing a runtime&#x20;
```

Comment:
Better to change to `Testing a runtime upgrade` or `Testing runtime upgrades`, no?